web/LICENSE.bin

```diff
@@ -203,15 +203,20 @@
 
    This product bundles various third-party components also under the
    Apache Software License 2.0.
-   
+
    Apache Zeppelin
    ./web/WEB-INF/web.xml
 
    @swc/helpers
+   typescript
 
    This product bundles various third-party components also under the
    MIT license.
 
+   MIT License Copyright (c) 2020-present, Vben
```

Comment:
@justinmclean Is this the correct format for adding it?

Reply:
you need to list the product name not the license and copyright

Reply:
Retained only Vben